cmd/bakery-keygen/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"os"
+
+	"gopkg.in/macaroon-bakery.v0/bakery"
+)
+
+var format = flag.String("f", "json", "output format (json, go)")
+
+func main() {
+	flag.Parse()
+	kp, err := bakery.GenerateKey()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "cannot generate key: %s\n", err)
+		os.Exit(1)
+	}
+	switch *format {
```

Comment:
I'd suggest we keep things ultra simple here and just use JSON.
No particular need to make the output format configurable, I think.
Then we don't need to parse arguments at all.